docs/roadmap.md

```diff
@@ -1,43 +0,0 @@
-# DocFX v3 Roadmap
```

Comment:
>Roadmap [](start = 11, length = 7)

place a link to the project dashboard rather then delete?

Reply:
README.md is updated to point to project dashboard.

---
In reply to: [181612382](https://github.com/dotnet/docfx/pull/2634#discussion_r181612382) [](ancestors = 181612382)